src/libmongoc/tests/unified/runner.c

```diff
@@ -1106,14 +1135,30 @@ test_check_expected_events_for_client (test_t *test,
    event_t *eiter = NULL;
    uint32_t expected_num_events;
    uint32_t actual_num_events = 0;
+   char *event_type = NULL;
 
    bp = bson_parser_new ();
    bson_parser_utf8 (bp, "client", &client_id);
    bson_parser_array (bp, "events", &expected_events);
+   bson_parser_utf8_optional (bp, "eventType", &event_type);
    if (!bson_parser_parse (bp, expected_events_for_client, error)) {
       goto done;
    }
 
+   if (event_type) {
+      if (0 == strcmp (event_type, "cmap")) {
+         /* TODO: (CDRIVER-3525) Explicitly ignore cmap events until CMAP is
+          * supported. */
+         ret = true;
+         goto done;
```

Comment:
This may overlap with my previous question but would it be preferable to error here and require skipping the tests?

Reply:
I have a preference for skipping the CMAP assertions and still running the test. They won't be testing as much, but it seems better to run what we can rather than skip the full test.